### PR TITLE
seismology: Add seismic wave

### PR DESCRIPTION
**Pull request by core-man**
Addresses #19.

### CODE REVIEW

File: source/seismology/seismic-wave.rst [hunk 1]
```diff
@@ -1,2 +1,212 @@
...
+.. figure:: body-wave-propagation.jpg
+   :alt: 体波的介质运动方向以及波传播方向
+   :width: 80%
+   :align: center
+
+   体波的介质运动方向以及波传播方向。
+   引自《\ `Introduction to Seismology <https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742>`__\ 》（第三版）
+   图 3.2。
+
+地球分层
```
Comment (core-man) [inline source/seismology/seismic-wave.rst:35]: I am think whether to move this subsection to the introduction page.

File: source/seismology/seismic-wave.rst [hunk 2]
```diff
@@ -1,2 +1,212 @@
...
+因此，S 波是一种横波（transverse wave），即介质的运动方向与波传播的方向垂直的波。
+
+.. figure:: body-wave-propagation.jpg
+   :alt: 体波的介质运动方向以及波传播方向
+   :width: 80%
+   :align: center
+
+   体波的介质运动方向以及波传播方向。
+   引自《\ `Introduction to Seismology <https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742>`__\ 》（第三版）
+   图 3.2。
```
Comment (seisman) [inline source/seismology/seismic-wave.rst:33]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

You may want to read this section to see how to reuse the definition of external links.

File: source/seismology/seismic-wave.rst [hunk 3]
```diff
@@ -1,2 +1,212 @@
 地震波
```
[the remark this answers is not in the document]
Comment (core-man) [reply source/seismology/seismic-wave.rst:1]: Done in 7ed5f18.

File: source/seismology/seismic-wave.rst [hunk 4]
```diff
@@ -1,2 +1,212 @@
...
+虽然体波同时由震源激发，但 P 波和 S 波的传播速度不同，S 波速度大约是 P 波速度的一半。
+P 波和 S 波传播速度也随深部而变化：
+
+- 在 Moho 面附近，地震波猛然增加
+- 在地幔中，410 公里和 660 公里附近存在两个速度间断面，地震波速度在间断面附近突然增加
+
+  .. note::
+  
+     这两个间断面分别称为 410 间断面（410-km discontinuity）和 660 间断面（660-km discontinuity），
+     他们之间的区域称为地幔转换带（mantle transition zone）。
```
[the remark this answers is not in the document]
Comment (core-man) [reply source/seismology/seismic-wave.rst]: Done in 79d766f.

File: source/seismology/seismic-wave.rst [hunk 5]
```diff
@@ -1,2 +1,212 @@
...
+   :align: center
+
+   地球内部 P 波速度、S 波速度和密度（来自 Preliminary Reference Earth Model (PREM)）。
+   引自《\ `Introduction to Seismology <https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742>`__\ 》（第三版）
+   图 1.1。
+
+体波波速
+^^^^^^^^
+
+虽然体波同时由震源激发，但 P 波和 S 波的传播速度不同，S 波速度大约是 P 波速度的一半。
```
[the remark this answers is not in the document]
Comment (core-man) [reply source/seismology/seismic-wave.rst]: Done in 587477e.

File: source/seismology/seismic-wave.rst (hunk 2 above)
Comment (core-man) [reply source/seismology/seismic-wave.rst]: Done in ff1f981.

File: source/seismology/seismic-wave.rst [hunk 6]
```diff
@@ -1,2 +1,211 @@
...
+
+.. figure:: body-wave-propagation.jpg
+   :alt: 体波的介质运动方向以及波传播方向
+   :width: 80%
+   :align: center
+
+   体波的介质运动方向以及波传播方向。引自《\ `Introduction to Seismology`_\ 》（第三版）图 3.2。
+   
+.. _Introduction to Seismology: https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742
+
```
Comment (seisman) [inline source/seismology/seismic-wave.rst:41]: Personally, I prefer to move the link definition to the end of this file, and also add the link definitions for "An Introduction to Seismology, Earthquakes, and Earth Structure" and "An Introduction to the Theory of Seismology" to make the figure caption shorter.

Comment (core-man) [reply source/seismology/seismic-wave.rst:41]: Okay

File: source/seismology/seismic-wave.rst (hunk 1 above)
Comment (seisman) [reply source/seismology/seismic-wave.rst:42]: I think we can keep it here now.

File: source/seismology/seismic-wave.rst (hunk 6 above)
Comment (core-man) [reply source/seismology/seismic-wave.rst]: Done in 41d56d3.

**Comment by core-man** [comment]
I want to and the first figure from https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-wave/intro.html. Maybe at the begining so that we could have a waveform on this page?

**Comment by seisman** [comment]
> I want to add the first figure from https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-wave/intro.html. Maybe at the begining so that we could have a waveform on this page. How do you think?

Sounds good

**Comment by core-man** [comment]
> > I want to add the first figure from https://migg-ntu.github.io/SeisTomo_Tutorials/seismology/seismic-wave/intro.html. Maybe at the begining so that we could have a waveform on this page. How do you think?
> 
> Sounds good

Added in e4c9e61.

**Review by seisman — APPROVED** [review]

**Pull request merged**